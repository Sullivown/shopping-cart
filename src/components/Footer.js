import React from 'react';

function Footer() {
	return (
		<footer>
			<div>
				Icons by <a href='https://icons8.com'>Icons8</a>
			</div>
			<div>AdventureMart Â©</div>
		</footer>
	);
}

export default Footer;
