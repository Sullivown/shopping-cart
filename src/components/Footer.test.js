import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import Footer from './Footer';

describe('Test Footer component', () => {
	it('renders', () => {
		const footerTestRenderer = TestRenderer.create(<Footer />).toJSON();
		expect(footerTestRenderer).toMatchSnapshot();
	});
});
