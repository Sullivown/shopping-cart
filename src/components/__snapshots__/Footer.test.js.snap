// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component renders 1`] = `
<footer>
  <div>
    Icons by 
    <a
      href="https://icons8.com"
    >
      Icons8
    </a>
  </div>
  <div>
    AdventureMart Â©
  </div>
</footer>
`;
